### ISSUE TITLE
Prot Paladin talent: Spiritual Attunement not working

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#1502]
i'm sorry but wowhead is outdated and the mania armory doesn't work for me so i'll use another database for link 
Spiritual attunement (https://wotlk.evowow.com/?spell=31785) talent is supposed to give your paladin mana when healed by other friendly targets.
Issue: currently not receiving any mana at all
Tested with a group member shaman's healing wave, 1000+ healing done (not overhealing) and zero mana received, Relogged, reloaded, the issue is constant and present on both of my paladins. (the other pally tested with a resto druid)

**Comment by captainloozer** [comment]
Hi Dranah, thank you for your report!

Unfortunately, I was not able to reproduce this on the test realm.  During my test, I found when healing a paladin with the spiritual attunement talent(31785), at full health it does not produce mana, which is intended. I also tested with the paladin at low health and it indeed gave the mana back as intended.

![satestwithlowhealth](https://user-images.githubusercontent.com/9828794/29463902-d63344d2-83f9-11e7-9729-5f056818f38e.PNG)
